test/parallel/test-net-server-listen-options.js

```diff
@@ -15,6 +15,10 @@ function close() { this.close(); }
   // Test listen({port})
   net.createServer().listen({ port: 0 })
     .on('listening', common.mustCall(close));
+  // Test listen(host, port}) on ipv4
+  net.createServer().listen({ host: '127.0.0.1', port: '3000' }).on('listening', common.mustCall(close));
+  // Test listen(host, port}) on ipv6
+  net.createServer().listen({ host: '::1', port: '3000' }).on('listening', common.mustCall(close));
```

Comment:
any chance we can add a happy path for a bit complex example like :

```js
net.createServer().listen({ host: 'fe80::1%eth0', port: '3000' }).on('listening', common.mustCall(close));
```

Reply:
I thought about it, but not sure if CI will complain about the zone-index, as not all VMs could have `eth0`

I can give it a try, but might be better for `%1` to pointing to the first network interface

Reply:
not blocking, could do in a separate PR